src/doc/src/reference/environment-variables.md

```diff
@@ -231,7 +231,7 @@ corresponding environment variable is set to the empty string, `""`.
 * `CARGO_PKG_RUST_VERSION` — The Rust version from the manifest of your package.
   Note that this is the minimum Rust version supported by the package, not the
   current Rust version.
-* `CARGO_CRATE_NAME` — The name of the crate that is currently being compiled.
+* `CARGO_CRATE_NAME` — The name of the crate that is currently being compiled, it is the [cargo target] with the `-` converted to `_`, such as the the name of the library, binary, example, integration test, or benchmark.
```

Comment:
Prefer capitalized by the way.

```suggestion
* `CARGO_CRATE_NAME` — The name of the crate that is currently being compiled. It is the name of the [Cargo target] with `-` converted to `_`, such as the name of the library, binary, example, integration test, or benchmark.
```

Reply:
Updated Rebased accordingly. Thank you!

Reply:
Thanks for the update. It's still not correct.

- "It is…" should start a new sentence.
- "**the name** of Cargo taget" instead of "the Cargo target"
- No "**the**" in "with `-` converted to `_`"
- Redundant **then** in "such as the the name of…"

Please see my suggestion above, though I am not a grammar expert 😜.

Reply:
BTW, the link reference `[Cargo target]` should be capitalized as well.

Reply:
Thank you :)